### PR TITLE
SMP-1836: add perf override file for cd, ci and ff

### PR DESCRIPTION
**Pull request by kapilgarg1996**

### CODE REVIEW

File: src/harness/override-perf-ci-cd-ff.yaml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      ORCHESTRATION_POOL_CORE_SIZE: "40"
+      ORCHESTRATION_POOL_MAX_SIZE: "300"
+      PMS_SDK_EXECUTION_POOL_CORE_SIZE: "40"
+      PMS_SDK_EXECUTION_POOL_MAX_SIZE: "300"
+      PMS_SDK_ORCHESTRATION_EVENT_POOL_CORE_SIZE: "40"
+      PMS_SDK_ORCHESTRATION_EVENT_POOL_MAX_SIZE: "300"
+      PIPELINE_EXECUTION_POOL_CORE_SIZE: "40"
+      PIPELINE_EXECUTION_POOL_MAX_SIZE: "300"
+      ORCHESTRATION_VISUALIZATION_POOL_CORE_SIZE: "30"
+      ORCHESTRATION_VISUALIZATION_POOL_MAX_SIZE: "50"
```
Comment (siraj-io) [inline src/harness/override-perf-ci-cd-ff.yaml:69]: this is additional vs apple

File: src/harness/override-perf-ci-cd-ff.yaml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    resources:
+      limits:
+        cpu: 4
+        memory: 6Gi
+      requests:
+        cpu: 4
+        memory: 6Gi
+    additionalConfigs:
+      REDIS_NETTY_THREADS: "32"
+  log-service:
```
Comment (siraj-io) [inline src/harness/override-perf-ci-cd-ff.yaml:104]: log service configuration is different than https://gist.github.com/kapilgarg1996/c3dc07dd4360c9bf9c9d93fa3bede532#file-gistfile1-txt-L51-L59

**Review by siraj-io — CHANGES_REQUESTED** [review]

File: src/harness/override-perf-ci-cd-ff.yaml (hunk 1 above)
Comment (siraj-io) [reply src/harness/override-perf-ci-cd-ff.yaml:69]: ORCHESTRATION_VISUALIZATION_POOL_MAX_SIZE: "50"

Comment (kapilgarg1996) [reply src/harness/override-perf-ci-cd-ff.yaml:69]: this is the one which was missing in gist as well. which was causing pipeline service to fail.

File: src/harness/override-perf-ci-cd-ff.yaml (hunk 2 above)
Comment (kapilgarg1996) [reply src/harness/override-perf-ci-cd-ff.yaml:104]: yeah, i will update the gist

File: src/harness/override-perf-ci-cd-ff.yaml (hunk 1 above)
Comment (siraj-io) [reply src/harness/override-perf-ci-cd-ff.yaml:69]: ok

Comment (siraj-io) [reply src/harness/override-perf-ci-cd-ff.yaml:69]: we are using ORCHESTRATION_VISUALIZATION_POOL_MAX_SIZE: "100" in smp-perf cluster, I think it should be 100

**Review by siraj-io — CHANGES_REQUESTED** [review]

**Review by siraj-io — APPROVED** [review]

**Pull request merged**